testing/scenario_app/lib/src/locale_initialization.dart

```diff
@@ -66,4 +68,42 @@ class LocaleInitialization extends Scenario {
       )).build()
     );
   }
+
+  /// Handle taps.
+  ///
+  /// Send changing information via semantics on each successive tap.
+  @override
+  void onPointerDataPacket(PointerDataPacket packet) {
+    String label;
+    switch(tapCount) {
+      case 0: label = window.platformResolvedLocale.toString(); break;
```

Comment:
Also, to make sure, it keeping track of the `_tapCount` solely to make sure we set the semantics label with the platform resolve locale on the first tap? Is there a reason we want it to only set it the first time?

Reply:
I expect to extend this in the future with further cases, and this allows easy extension by just adding an extra case here and sending another tap.